tests/NRedisStack.Tests/Bloom/BloomTests.cs

```diff
@@ -11,7 +11,7 @@ public class BloomTests : AbstractNRedisStackTest, IDisposable
 
     public void Dispose()
     {
-        redisFixture.Redis.GetDatabase().KeyDelete(key);
+        redisFixture.Redis.GetDatabase().ExecuteBroadcast("FLUSHALL");
```

Comment:
Is there any way to have a general dispose? Since these all do the same thing a single dispose that uses FLUSHALL would make sense to me.